terraform/main.tf

```diff
@@ -0,0 +1,52 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+# data bucket lambda writes to
+resource "aws_s3_bucket" "data" {
+  acl = "public-read"
+  # The bucket name is hardcoded into the app
+  # Make sure to change there if you change me!
+  bucket = "codefornashville-inclucivics-c9b520"
+
+  cors_rule {
+    allowed_methods =["GET"]
+    allowed_origins = ["*"]
```

Comment:
Did you add this `cors` rule? I don't remember it and it makes me nervous.

Reply:
Muahahahahaha.  I think I figured out how to let localhost through after reading a little more - will update.